### PR TITLE
TMGR-27: fix ui issues for desktop app

### PR DESCRIPTION
**Pull request by acrossoffwest**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,11 @@
 	"scripts": {
 		"serve": "quasar dev",
 		"build": "quasar build",
-		"build-linux": "quasar build -m electron --fix",
-		"build-mas": "quasar build -m electron --fix -T mas",
-		"build-windows": "quasar build -m electron --fix -T win32",
-		"serve-windows": "quasar dev --mode electron --fix -T win32",
-		"serve-mas": "quasar dev --mode electron --fix -T mas",
+		"build:linux": "quasar build -m electron --fix",
+		"build:mas": "PYTHON_PATH=/Users/iuriikarpov/.pyenv/shims/python quasar build -m electron --fix -T mas -b builder",
```
Comment (savayer) [inline package.json:11]: not good )

File: src/assets/styles/index.scss [hunk 2]
```diff
@@ -147,3 +147,13 @@ button {
...
+
+.dark .dark\:bg-neutral-900 {
+	--tw-bg-opacity: 1;
+	background-color: rgb(23 23 23/var(--tw-bg-opacity));
+}
+
+.bg-neutral-100 {
+	--tw-bg-opacity: 1;
+	background-color: rgb(245 245 245/var(--tw-bg-opacity));
+}
```
Comment (savayer) [inline src/assets/styles/index.scss:159]: don't rewrite TW styles

File: src/App.vue [hunk 3]
```diff
@@ -3,35 +3,36 @@
...
-				icon="close"
-				@click="closeApp"
-			/>
+				<q-btn
+					class="q-electron-drag--exception"
+					dense
+					flat
+					icon="close"
+					@click="closeApp"
+				/>
```
Comment (savayer) [inline src/App.vue:34]: Let's not use quasar components if they don't give us a profit.

**Review by savayer — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (acrossoffwest) [reply package.json]: fix

File: src/assets/styles/index.scss (hunk 2 above)
Comment (acrossoffwest) [reply src/assets/styles/index.scss]: ok, fix

**Pull request merged**